python_modules/dagster/dagster/_core/definitions/auto_materialize_rule.py

```diff
@@ -1058,39 +1055,14 @@ def decision_type(self) -> AutoMaterializeDecisionType:
 
     @property
     def description(self) -> str:
-        return "required parent partitions do not exist"
+        return self.to_asset_condition().description
 
-    def evaluate_for_asset(
-        self, context: AssetConditionEvaluationContext
-    ) -> "AssetConditionResult":
-        from .asset_condition.asset_condition import AssetConditionResult
-
-        asset_partitions_by_evaluation_data = defaultdict(set)
-
-        subset_to_evaluate = (
-            context.candidates_not_evaluated_on_previous_tick_subset
-            | context.candidate_parent_has_or_will_update_subset
+    def to_asset_condition(self) -> AssetCondition:
```

Comment:
probably missed this somewhere else, but how are we forking between to_asset_condition and evaluate_for_asset ?